runtime/src/main/java/io/quarkiverse/semantickernel/SemanticKernelClientProducer.java

```diff
@@ -33,11 +33,14 @@ public OpenAIAsyncClient produceOpenAIAsyncClient() throws ConfigurationExceptio
         } else {
             // OPEN AI
             if (semanticKernelConfiguration.client().get().openai().isPresent()) {
+                // TBV Just quick and dirty hacks which need to be properly engineered
```

Comment:
That's ok. I've submitted a PR on Semantic Kernel so we could have access to these properties. It will be refactored once the PR is merged

Reply:
Nice, then... is it ok  to resolve this conversation?